pkg/util/losetup/losetup_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2024 ScyllaDB.
+
+package losetup
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/scylladb/scylla-operator/pkg/util/exectest"
+	testingexec "k8s.io/utils/exec/testing"
+)
+
+func TestListLoopDevices(t *testing.T) {
+	t.Parallel()
+
+	tt := []struct {
+		name                string
+		commands            []exectest.Command
+		expectedLoopDevices []*LoopDevice
+		expectedError       error
+	}{
+		{
+			name: "losetup fails with random error",
+			commands: []exectest.Command{
+				{
+					Cmd:    "losetup",
+					Args:   []string{"--all", "--list", "--json"},
+					Stdout: []byte("stdout error"),
```

Comment:
tiny nit: i suppose since the error is within stderr, "stdout output" or something like that might be more fitting here

Reply:
done